Badge/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="badge-container">
+      <img src="/Badge/shopping-cart (1).png" alt="" />
```

Comment:
**FIX:** Give value to the "alt" attribute as that helps for accessibility.

Reply:
Sure vishal thanks for pointing these out i will fix these and commit